pkg/operation/botanist/controlplane.go

```diff
@@ -1079,52 +1084,99 @@ func (b *Botanist) DeployETCD(ctx context.Context) error {
 		"storageCapacity": b.Seed.GetValidVolumeSize("10Gi"),
 	}
 
-	etcd, err := b.InjectSeedShootImages(etcdConfig, common.ETCDImageName)
+	imageNames := []string{
+		common.ETCDImageName,
+		common.ETCDBackupRestoreImageName,
+	}
+
+	values, err := b.InjectSeedShootImages(etcdConfig, common.ETCDImageName)
+	if err != nil {
+		return err
+	}
+
+	images, err := imagevector.FindImages(b.ImageVector, imageNames, imagevector.RuntimeVersion(b.SeedVersion()), imagevector.TargetVersion(b.ShootVersion()))
 	if err != nil {
 		return err
 	}
 
+	values["prefix"] = common.GenerateBackupEntryName(b.Shoot.Info.Status.TechnicalID, b.Shoot.Info.Status.UID)
+
 	for _, role := range []string{common.EtcdRoleMain, common.EtcdRoleEvents} {
-		etcd["role"] = role
+		values["global"] = map[string]interface{}{
```

Comment:
Why do you need this "global" key? Can't you make `images` a top-level key? Then you don't need this `ImageMapToValues` function, or?

Reply:
makes sense. Done.